lib/queries.js

```diff
@@ -64,9 +64,10 @@ export function getFilterQuery(table, filters = {}, params = []) {
 
     switch (key) {
       case 'url':
-        if (table === 'session' || table === 'pageview') {
-          arr.push(`and ${table}.${key}=$${params.length + 1}`);
+        if (table === 'session' || table === 'pageview' || table === 'event') {
+          arr.push(`and (${table}.${key}=$${params.length + 1} or ${table}.${key} like $${params.length + 2})`);
```

Comment:
I don't like the additional OR here. Maybe we can just use the like query instead.

Reply:
If we simple use `like` to query `/foo%` we will also get results under `/foo/bar` or is there another match?